app/services/evss_claim_service.rb

```diff
@@ -123,6 +124,20 @@ def record_workaround(task, claim_id, job_id)
                         })
   end
 
+  def record_evidence_submission(claim_id, job_id, tracked_item_id)
+    user_account_id = @user.user_account_uuid
+    user_account = UserAccount.find(@user.user_account_uuid)
+    job_class = self.class
+    upload_status = 'pending'
+    evidence_submission = EvidenceSubmission.build(claim_id:,
```

Comment:
Is there a reason to use `.build` here instead of `.new`?

Reply:
@jerekshoe Yeah, I don't want it to save until after assigning the assocation. `.new` would save it right away.